src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -362,7 +366,15 @@ def fuzz(self,
            additional_args=None,
            extra_env=None):
     """LibFuzzerCommon.fuzz override."""
-    return self._test_qemu_ssh()
+    self._test_qemu_ssh()
+    # Doing a timeout, to work around process-exit issue?
+    self.fuzzer.run([])
```

Comment:
could you clarify what the issue is here?

Reply:
ah! that issue's obsolete; was only an issue while debugging. removed comment